hack/bootstrap-ginkgo.sh

```diff
@@ -4,12 +4,16 @@ set -e
 
 source $(dirname "$0")/common.sh
 
-_out/tests/ginkgo version
+FOLDERS="${KUBEVIRT_DIR}/staging/src/kubevirt.io/ ${KUBEVIRT_DIR}/pkg/ ${KUBEVIRT_DIR}/cmd/ ${KUBEVIRT_DIR}/tests/framework/"
```

Comment:
nit: Could be nice nave the directories ordered by name:
```suggestion
FOLDERS="${KUBEVIRT_DIR}/cmd/ ${KUBEVIRT_DIR}/pkg/ ${KUBEVIRT_DIR}/staging/src/kubevirt.io/   ${KUBEVIRT_DIR}/tests/framework/"
```

Reply:
Fixed